.env.example

```
@@ -14,9 +14,10 @@ REDIS_HOST=
 REDIS_PORT=
 
 # Config Server
-CONFIG_USERNAME=
-CONFIG_PASSWORD=
-CONFIG_URL=
+CONFIG_USERNAME=      # 팀 공유값: festie
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)